videoanalyst/model/task_head/taskhead_impl/track_head.py

```diff
@@ -83,7 +83,7 @@ def __init__(self):
         self.cls_convs = []
         self.bbox_convs = []
 
-    def forward(self, c_out, r_out):
+    def forward(self, c_out, r_out, x_size):
```

Comment:
Consider assigning a default value (e.g. -1 indicating "not given") to _x_size_ so that the signature won't change in case that _input_size_adapt_ is False?